Source/DafnyCore/Coco/Scanner.frame

```diff
@@ -415,48 +411,6 @@ public class Scanner {
         if (ch == '\r' && buffer.Peek() != '\n') ch = EOL;
         if (ch == EOL) {
           line++; col = 0;
-        } else if (false && ch == '#' && col == 1) {
```

Comment:
This code for recognizing `#line` pragmas was inherited from Boogie. We're currently not using it in Dafny, as evidenced by the `false &&` on L418. If the code only looked for `#line` (and not `#` followed by other things), then it does not interfere with otherwise legal Dafny programs. Still, since we're not using `#line`, it's fine with me if we remove the code. If we ever wanted this `#line` functionality in the future, it would be easy enough to copy the code from Boogie again or to write the code anew.